internal/storage/sqlite/classroom_store.go

```
@@ -0,0 +1,800 @@
+	defer s.finishTx(tx, &resultErr, "list classroom campuses")
+	rows, err := tx.QueryContext(ctx, `
+SELECT id,name,source_revision FROM classroom_campuses
+WHERE app_id=? AND source_revision=? ORDER BY name,id`, appID, metadata.Revision)
+	if err != nil {
+		return classroom.CampusSnapshot{}, fmt.Errorf("query classroom campuses: %w", err)
+	}
+	campuses := []classroom.Campus{}
+	for rows.Next() {
+		var campus classroom.Campus
+		if err := rows.Scan(&campus.ID, &campus.Name, &campus.SourceRevision); err != nil {
+			rows.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rows.Close` is not checked

(errcheck)